quantities/prefixes/Build.cs

```diff
@@ -0,0 +1,14 @@
+using Quantities.Numerics;
+using static Quantities.Extensions;
+
+namespace Quantities.Prefixes;
+
+internal static class Build<TPrefix>
```

Comment:
This class is a bit of an anoyance, as we first need to get hold of a conversion polynomial before we can do any conversion. So this is just a "helper class" that encapsulates that fact away behind a reasonably easy to understand api. (usage see: [BinaryPrefix.cs](quantities/prefixes/BinaryPrefix.cs))

Reply:
Since we made everything depend on the SI units, it was clear that somewhere in the code, all conversions & builds would have to go through an SI unit before being converted to the target unit. The footprint of this class is small, so it seems okay to me 😊